src/pushsource/_impl/utils/containers/request.py

```diff
@@ -302,11 +302,17 @@ def inspect(registry, repo, digest, token=None):
     token = token or AuthToken()
 
     manifest_type, digest, manifest = get_manifest(
-        registry, repo, digest, manifest_types=[MEDIATYPE_SCHEMA2_V2_LIST], token=token
+        registry, repo, digest, manifest_types=[MEDIATYPE_SCHEMA2_V2], token=token
     )
     if manifest_type == MEDIATYPE_SCHEMA2_V2:
+        print("v2 sch2")
```

Comment:
Please remove the prints.  You could replace them with some debug logs if the info might be useful.